script/generate-component-index

```diff
@@ -23,3 +24,22 @@ do
 done
 
 echo "};" >> $target
+
+# Build sass index
+target=./src/styles/components.scss
+
+echo "// Generated by $0" > $target
+echo "@import './colors.scss';" >> $target
+echo "" >> $target
+
+for f in $( find src/components -name '*.scss' | sort )
+do
+    base=$( basename "$f" )
+    echo "@import '../components/$base';" >> $target
+done
+
+for f in $( find src/pages -name '*.scss' | sort )
+do
+    base=$( basename "$f" )
+    echo "@import '../pages/$base';" >> $target
+done
```

Comment:
Rather than repeating, why not just all of `src`? In which case we could call it something instead of `components.scss`.